src/QirRuntime/samples/StandaloneInputReference/qir-driver.cpp

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <iostream>
+#include "CLI11.hpp"
+#include "QuantumApi_I.hpp"
+#include "SimFactory.hpp"
+#include "QirContext.hpp"
+
+using namespace Microsoft::Quantum;
+
+extern "C" void Quantum__StandaloneSupportedInputs__ExerciseInputs__body( // NOLINT
+    int64_t anInt,
+    double aDouble);
+
+int main(int argc, char *argv[])
+{
+    try{
+        CLI::App app("QIR Standalone Entry Point Inputs Reference");
+
+        // Add the --simulator-output and --operation-output options.
```

Comment:
What is the difference between the simulator and operation output?

Reply:
The simulation output is the one produced by calls to the `Message` Q# API. The operation output is what the Q# entry-point operation returns. In this case it is an `Int` but it can be a `Result`, an array of `Results` or whatever the a Q# entry-point operation can return.